lib/middlewares/responseGenerator.js

```diff
@@ -175,19 +193,31 @@ function manageIncomingResponse(id, value) {
 
     var data = synchronousRequests[id];
 
-    if (!data.request.configurationValues) {
-        data.request.configurationValues = {};
-    }
+    if (data && data.request && value && value.length === 1 && value[0].name && value[0].name === 'lazy_op_result') {
+        if (!data.request.configurationValues) {
+            data.request.configurationValues = {};
+        }
 
-    data.request.configurationValues['op_result'] = value[0];
-    processAsynchronousRequest(data.request, data.response, data.callback);
+        data.request.configurationValues['lazy_op_result'] = value[0];
```

Comment:
For the time being we are using `_` as the separator in the attribute names for entities... :) But let's see what's @dmoranj 's opinion on this :)